packages/controls/src/utils.ts

```diff
@@ -1,65 +1,15 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import * as _ from 'underscore';
+export {
+    uuid, WrappedError, resolvePromisesDict
+} from '@jupyter-widgets/base';
 
-/**
- * http://www.ietf.org/rfc/rfc4122.txt
- */
-export
-function uuid(): string {
-    let s = [];
-    const hexDigits = '0123456789ABCDEF';
-    for (let i = 0; i < 32; i++) {
-        s[i] = hexDigits.substr(Math.floor(Math.random() * 0x10), 1);
-    }
-    s[12] = '4';  // bits 12-15 of the time_hi_and_version field to 0010
-    s[16] = hexDigits.substr((s[16] & 0x3) | 0x8, 1);  // bits 6-7 of the clock_seq_hi_and_reserved to 01
+import {
+    WrappedError
+} from '@jupyter-widgets/base';
```

Comment:
Note: This is now imported twice.

Reply:
Thanks. See #1808

Reply:
Oh wait. it is not imported twice. It's exported once, and imported once.

Reply:
Oh! Sorry for the noise then!